src/app/routes/connector-ui/contract-agreement-page/contract-agreement-transfer-dialog/contract-agreement-transfer-dialog.component.ts

```diff
@@ -133,20 +133,23 @@ export class ContractAgreementTransferDialogComponent implements OnDestroy {
       };
     }
 
-    let transferRequestProperties =
+    let transferProcessProperties =
       this.httpRequestParamsMapper.encodeHttpProxyTransferRequestProperties(
         this.data.asset,
         value,
       );
 
     let dataSinkProperties =
-      this.dataAddressMapper.buildDataAddressProperties(value).properties ?? {};
+      {
+        properties:
+          this.dataAddressMapper.buildDataAddressProperties(value).properties,
+      } ?? {};
```

Comment:
dataSinkProperties should be of type `Record<string, string>`, not of type `DataAddressDto` (unnest the nested properties object)